source/help/apps/desktop-guide.rst

```diff
@@ -4,32 +4,32 @@ Using Mattermost Desktop App
 
 Mattermost desktop applications are available for Windows, Mac and Linux operating systems. 
 
-You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up.
+You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up and for troubleshooting tips.
 
 To view the latest updates, please see our `changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_.
 
-Below are a few tips to get you started and configuring your experience on the desktop app:
+Below are a few tips to get you started and to configure your experience on the desktop app:
 
  - `Team Management <https://docs.mattermost.com/help/apps/desktop-guide.html#id1>`_
  - `App Options <https://docs.mattermost.com/help/apps/desktop-guide.html#id2>`_
  - `Menu Bar <https://docs.mattermost.com/help/apps/desktop-guide.html#id3>`_
- - `Troubleshooting <https://docs.mattermost.com/help/apps/desktop-guide.html#id4>`_
 
 Team Management
 ---------------------------------------------------------------------
 
 You can connect to multiple Mattermost servers from a single interface on the desktop app.
 
-Team Management appears in the Settings page, available from the `File` menu under `Settings`.
+The Team Management section allows you to add, edit and remove servers and teams.
 
 Adding servers and teams
```

Comment:
We have to be careful because it's easy for people to think that they can create a new team by following this procedure, when really all they're doing is adding an existing team to a list.

But then it's even more misleading because this doesn't actually add a team to the list. It adds a server and a name for the tab. The tab might have the same name as a team, but that would be a coincidence. You can have more than one team under the tab.

Maybe we need to revisit this doc outside the limits of a PR?

![image](https://cloud.githubusercontent.com/assets/20325173/22044134/ee207558-dcc7-11e6-894a-52a29afa730a.png)